src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -865,8 +869,31 @@ public static String getBucketNameForLocation(String bucketPrefix, String locati
    */
   public static Map<String, String> getJobTags(String jobType) {
     Map<String, String> labels = new HashMap<>();
-    labels.put("job_source", "cdap");
-    labels.put("type", jobType);
+    labels.put(BQ_JOB_SOURCE_TAG, "cdap");
+    labels.put(BQ_JOB_TYPE_TAG, jobType);
+    return labels;
+  }
+
+  /**
+   * Get tags for a BigQuery Job.
+   * @param jobType the job type to set in the labels.
+   * @param userDefinedTags user defined tags to be added to the labels.
+   * @return map containing labels for a BigQuery job launched by this plugin.
+   */
+  public static Map<String, String> getJobTags(String jobType, String userDefinedTags) {
```

Comment:
Can you please rename this and method on line 870 to `getJobLabels`? Tags in itself a separate resource in GCP, do not want confuse it with labels.

Reply:
This function was not written by me will it be okay to change the name here ?

Reply:
should be fine. It is not an API so it does not need to be backward compatible.

Reply:
Done !